components/automate-ui/src/app/modules/infra-proxy/cookbook-list/cookbooks-list.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="content-container">
+    <div class="container">
+        <main>
+            <chef-breadcrumbs>
+                <chef-breadcrumb [link]="['/infrastructure/chef-servers']">Chef Servers</chef-breadcrumb>
+                <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', org?.server_id]">Orgs</chef-breadcrumb>
+                {{ org?.name }}
+            </chef-breadcrumbs>
+            <chef-page-header>
+                <chef-heading>{{ org?.name }}</chef-heading>
+                <table>
+                    <thead>
+                        <tr class="detail-row">
+                            <th class="id-column">Name</th>
+                            <th class="id-column">Admin User</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr class="detail-row">
+                            <td class="id-column">{{ org?.name }}</td>
+                            <td class="id-column">{{ org?.admin_user }}</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <chef-tab-selector [value]="tabValue" (change)="onSelectedTab($event)">
+                    <chef-option value='cookbooks' data-cy="cookbooks-tab">Cookbooks</chef-option>
+                    <chef-option value='details' data-cy="details-tab">Details</chef-option>
+                </chef-tab-selector>
+            </chef-page-header>
+            <section class="page-body" *ngIf="tabValue === 'details'">
+                <form [formGroup]="updateOrgForm">
+                    <chef-form-field>
+                        <label>
+                            <span class="label">Name <span aria-hidden="true">*</span></span>
+                            <input chefInput formControlName="name" type="text" autocomplete="off"
+                                data-cy="update-org-name">
+                        </label>
+                        <chef-error
+                            *ngIf="(updateOrgForm.get('name').hasError('required') || updateOrgForm.get('name').hasError('pattern')) && updateOrgForm.get('name').dirty">
+                            Display Name is required.
+                        </chef-error>
+                    </chef-form-field>
+                    <chef-form-field>
+                        <label>
+                            <span class="label">Admin User <span aria-hidden="true">*</span></span>
+                            <input chefInput formControlName="admin_user" type="text" autocomplete="off"
+                                data-cy="update-Org-admin-user">
```

Comment:
is this a typo?    thinking maybe it was supposed to be `data-cy="update-Org-admin-user"`, if not please disregard.